src/lib/utils/hooks/SynapseAPI/provenance/useGetActivityForEntity.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery, UseQueryOptions } from 'react-query'
+import { SynapseClient } from '../../..'
+import { SynapseClientError } from '../../../SynapseClientError'
+import { useSynapseContext } from '../../../SynapseContext'
+import { Activity } from '../../../synapseTypes/Provenance/Provenance'
+
+export function useGetActivityForEntity(
+  entityId: string,
+  versionNumber: number,
+  options?: UseQueryOptions<Activity, SynapseClientError, Activity>,
+) {
+  const { accessToken } = useSynapseContext()
+  return useQuery<Activity, SynapseClientError>(
+    ['getActivityForEntity', accessToken],
```

Comment:
Should include entityId, version number in query key

Reply:
I think I fixed, could you re-review?